docs/install/install-extensions.md

```diff
@@ -0,0 +1,278 @@
+
+```bash
+kubectl apply -f {{< artifact repo="eventing" file="eventing-sugar-controller.yaml" >}}
+```
+
+The Knative Eventing Sugar Controller will react to special labels and
+annotations and produce Eventing resources. For example:
+
+- When a Namespace is labeled with `eventing.knative.dev/injection=enabled`, the
+  controller will create a default broker in that namespace.
+- When a Trigger is annotated with `eventing.knative.dev/injection=enabled`, the
+  controller will create a Broker named by that Trigger in the Trigger's
+  Namespace.
+
+The following command enables the default Broker on a namespace (here
+`default`):
+
+```bash
+kubectl label namespace default eventing.knative.dev/injection=enabled
+```
+
+{{< /tab >}}
+
+{{% tab name="Github Source" %}}
+
+The following command installs the single-tenant Github source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-github" file="github.yaml" >}}
+```
+
+The single-tenant GitHub source creates one Knative service per GitHub source.
+
+The following command installs the multi-tenant GitHub source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-github" file="mt-github.yaml" >}}
+```
+
+The multi-tenant GitHub source creates only one Knative service handling all
+GitHub sources in the cluster. This source does not support logging or tracing
+configuration yet.
+
+To learn more about the Github source, try
+[our sample](../eventing/samples/github-source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Apache Camel-K Source" %}}
+
+The following command installs the Apache Camel-K Source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-camel" file="camel.yaml" >}}
+```
+
+To learn more about the Apache Camel-K source, try
+[our sample](../eventing/samples/apache-camel-source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Apache Kafka Source" %}}
+
+The following command installs the Apache Kafka Source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-kafka" file="source.yaml" >}}
+```
+
+To learn more about the Apache Kafka source, try
+[our sample](../eventing/samples/kafka/source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="GCP Sources" %}}
+
+The following command installs the GCP Sources:
+
+```bash
+# This installs both the Sources and the Channel.
+kubectl apply -f {{< artifact org="google" repo="knative-gcp" file="cloud-run-events.yaml" >}}
+```
+
+To learn more about the Cloud Pub/Sub source, try
+[our sample](../eventing/samples/cloud-pubsub-source/README.md).
+
+To learn more about the Cloud Storage source, try
+[our sample](../eventing/samples/cloud-storage-source/README.md).
+
+To learn more about the Cloud Scheduler source, try
+[our sample](../eventing/samples/cloud-scheduler-source/README.md).
+
+To learn more about the Cloud Audit Logs source, try
+[our sample](../eventing/samples/cloud-audit-logs-source/README.md).
```

Comment:
Since this story was to split existing content and not to edit it, I think fixing style related things like link text should be part of a separate issue.